episodes/designing-a-light-microscopy-experiment.md

```diff
@@ -0,0 +1,402 @@
+microscopy'. This is simply where the sample is illuminated by a bright light 
+from one side, and then imaged from the other. The issue with this method is 
+that it often produces low-contrast images, where biological structures are 
+hard to see. This is because biological structures are often quite transparent - 
+they don't absorb much light or differ much in their density. For this reason, 
+contrast agents/staining are often used to increase contrast (i.e. the addition 
+of dyes/chemicals that bind to specific structures).
+
+![The image above is Napari's Skin (RGB) sample image - it is a brightfield 
+image of a hematoxylin and eosin stained slide of dermis and epidermis.
+](fig/skin.png){alt="Screenshot of Napari's Skin sample image" width='70%'}
+
+To increase contrast (especially for unstained samples), widefield microscopes 
+often support ['phase-contrast'
+](https://www.microscopyu.com/techniques/phase-contrast/introduction-to-phase-contrast-microscopy) 
+or ['DIC - Differential Interference Contrast'
+](https://www.leica-microsystems.com/science-lab/microscopy-basics/differential-interference-contrast-dic/).
+
+![The image above is a phase gradient contrast image of some SH-SY5Y cells 
+(ZEISS Microscopy, [CC BY 2.0](https://creativecommons.org/licenses/by/2.0), 
+via [Wikimedia Commons
+](https://commons.wikimedia.org/wiki/File:SH-SY5Y_cells,_transmitted_light_phase_gradient_contrast_microscopy_with_ZEISS_Celldiscoverer_7_(30614936722).jpg)
+)](fig/phase-contrast.jpg){alt="Phase gradient contrast image of SH-SY5Y 
+cells" width='70%'}
+
+![The image above is a DIC image of some yeast cells (_Saccharomyces cerevisiae_) 
+from [Wikimedia Commons
+](https://commons.wikimedia.org/wiki/File:S_cerevisiae_under_DIC_microscopy.jpg)
+](fig/dic-example.jpg){alt="DIC image of some yeast cells - Saccharomyces 
+cerevisiae" width='50%'}
+
+Widefield microscopes can also be used for fluorescence microscopy. In 
+fluorescence microscopy, fluorescent labels are used that target specific 
+features (like the nucleus or cell membrane). These labels are excited by a 
+specific wavelength of light (the excitation wavelength), and then emit light of 
+a longer wavelength (the emission wavelength). By illuminating the sample with 
+light of the excitation wavelength, then detecting light of the emission 
+wavelength, a fluorescence microscope can image the biological structures that 
+the label is bound to.
+
+![The image above is a fluorescence microscopy image of some LLC-PK1 cells 
+(ZEISS Microscopy, [CC BY 2.0](https://creativecommons.org/licenses/by/2.0), 
+via [Wikimedia Commons
+](https://commons.wikimedia.org/wiki/File:Mitotic_LLC-PK1_cells,_fluorescence_microscopy_(23700644352).jpg)
+)](fig/fluorescence-example.jpg){alt="Fluorescence microscopy image of some 
+LLC-PK1 cells" width='70%'}
+
+The main issue with widefield microscopes is that, due to the whole sample being 
```

Comment:
So this is essentially compressing the third dimension down to two?

Reply:
I think so - you're detecting light from the focal plane + areas that are illuminated above and below it. So you're definitely incorporating some 3D information into your 2D image.